### ISSUE TITLE
keep getting messages that an ISBN already exists in the cataloge even though it most definately doesn't

### ISSUE THREAD

**Issue by hasselby** [Librarika/Issues#605]
Hi
i'm a librarian using the Liberika software at IES i Hässelby library - we have an account
i'm trying to add new books to our library using the manual add function
i keep getting warnings saying that the ISBN i'm trying to use already exists (i'm trying to add books manually using a list unique 10 digit ISBN stickers purchased separately) already exists in the catalog - even though it is not and is listed in Liberika under its own different ISBN
what should i do?

**Comment by rasel2001** [comment]
Are you still having this issue. If yes, please send us the ISBN numbers and we will look onto this.

**Comment by QueerLibrary** [comment]
We are having the same problem! I was trying to enter "The Transgender Child" (ISBN 9781573443180) and it told me that it had the same ISBN as "Luncheonette: A memoir" (0060728922). So I deleted both books from the system and reentered Luncheonette, with all the same info. Then I tried to enter The Transgender Child again and this time it told me it has the same ISBN as "Becoming a Man: Half a Life Story" which has an ISBN of 0060595647.

**Comment by timowen2** [comment]
I have always had this problem with manual additions, I get around it by not entering the ISBN

**Comment by srhglsn73** [comment]
I am having this problem also. This is happening repeatedly while I am attempting to add books manually. I have deleted books out of the system that say the isbn # belongs to them when it obviously does not. When I do that a different book will pop up with the same warning. This is very frustrating especially since the program is payed for.

**Comment by timowen2** [comment]
Yes it is an issue. Usually they fix these things fairly quickly for this sort of problem. We have been working around it for a long time by just ignoring the ISBN

Tim

Sent from my Huawei phone


-------- Original message --------

**Comment by LibraryMage** [comment]
Same here. I am having this error.

**Comment by rasel2001** [comment]
@LibraryMage Please share your library name & the ISBN number so that we can look onto this. 

Please note: This problem only occurs if that ISBN is already present in your library catalog.

**Comment by LibraryMage** [comment]
Hello,

Turkish Maarif Foundation Curriculum Library, https://tmfcurriculumlibrary.librarika.com

Librarika: Turkish Maarif Foundation Curriculum Library<https://tmfcurriculumlibrary.librarika.com/>
Turkish Maarif Foundation Curriculum Library is using Librarika, the free Integrated Library System (ILS) for managing its library online.You can also leverage this wonderful library information platform to manage your organization's library, family library or even your own personal library online.
tmfcurriculumlibrary.librarika.com
and the item that troubles me is 0740617289206...There are almost 200-300 more from this flashdisk which i need to catalog them.[cid:69feb2a4-a2f6-4c7b-8af7-28bad45caddf]
________________________________
Gönderen: rasel2001 <notifications@github.com>
Gönderildi: 13 Kasım 2020 Cuma 15:38
Kime: Librarika/Issues <Issues@noreply.github.com>
Bilgi: LibraryMage <volkanbostanci@windowslive.com>; Mention <mention@noreply.github.com>
Konu: Re: [Librarika/Issues] keep getting messages that an ISBN already exists in the cataloge even though it most definately doesn't (#605)


@LibraryMage<https://github.com/LibraryMage> Please share your library name & the ISBN number so that we can look onto this.

Please note: This problem only occurs if that ISBN is already present in your library catalog.

**Comment by rasel2001** [comment]
@LibraryMage As We mentioned earlier that this problem only occurs if same ISBN is already present in your library catalog. 

We have seen that a catalog item already present in your library with ISBN . The catalog item is **Kingston 32GB USB 2.0 Data Traveler 104, 30MB/s Read, 5MB/s Write (DT104/32GB)**. If the ISBN number does not belongs to that catalog item, please remove or update before you add more catalog.
